docs/components/sign-in.md

```diff
@@ -0,0 +1,221 @@
+# SignIn Component
+
+`SignIn` is a React higher order component that manages the user sign-in process, including handling status changes, user information updates, and error handling. It's designed to be flexible and customizable, allowing you to pass in custom components.
+
+## Props
+
+The `SignIn` component takes the following props:
+
+| Prop             | Type          | Default Value | Description |
+|------------------|---------------|---------------|-------------|
+| authStatus       | string        | 'Signed Out'  | The initial authentication status. |
+| connect          | object        | N/A           | The Greenruhm Connect SDK. In order for this component to work, it is necessary that the SDK is initialized and provided. |
+| ErrorComponent   | component     | N/A           | A component to display in case of an error. This prop will not be used in case `ViewComponent` is passed as well. |
```

Comment:
In all of these override cases, say:

If `ViewComponent` is passed, this component is ignored because the `ViewComponent` should handle this.